jupyter_kernel_mgmt/subproc/manager.py

```diff
@@ -42,44 +43,44 @@ def __init__(self, popen, files_to_cleanup=None, win_interrupt_evt=None):
         self.log = get_app_logger()
         self.kernel_id = str(uuid.uuid4())
 
-    def wait(self, timeout):
+    async def wait(self, timeout):
         """"""
         if timeout is None:
             # Wait indefinitely
-            self.kernel.wait()
+            await maybe_future(self.kernel.wait())
             return False
 
         if six.PY3:
```

Comment:
If we're going async/await on this code, we can probably drop the Python 2 branch.

Reply:
Good catch - thanks.